src/test/java/seedu/address/testutil/TypicalBorrowers.java

```diff
@@ -28,6 +28,8 @@
     public static final BorrowerId ID_FIRST_BORROWER = new BorrowerId("K0001");
     public static final BorrowerId ID_SECOND_BORROWER = new BorrowerId("K0002");
 
+    public static final BorrowerId ID_ALICE = new BorrowerId("K0001");
```

Comment:
Can we make us of the already declared `ID_FIRST_BORROWER` instead of creating a new `ID_ALICE`? Just a suggestion

Reply:
> Can we make us of the already declared `ID_FIRST_BORROWER` instead of creating a new `ID_ALICE`? Just a suggestion

Was going to use alice's ID and I thought using the name ID_ALICE feels clearer. But going to remove this and use getBorrowerId() instead anyway!